certora/applyHarness.patch

```diff
@@ -42,20 +42,84 @@ diff -druN Safe.sol Safe.sol
      }
  }
 diff -druN base/Executor.sol base/Executor.sol
---- base/Executor.sol	2023-11-10 09:50:31
-+++ base/Executor.sol	2023-11-20 10:28:47
-@@ -26,12 +26,8 @@
-         uint256 txGas
-     ) internal returns (bool success) {
-         if (operation == Enum.Operation.DelegateCall) {
--            /* solhint-disable no-inline-assembly */
--            /// @solidity memory-safe-assembly
--            assembly {
+--- base/Executor.sol	2023-12-06 16:00:31
++++ base/Executor.sol	2023-12-07 00:01:59
+@@ -29,8 +29,10 @@
+             /* solhint-disable no-inline-assembly */
+             /// @solidity memory-safe-assembly
+             assembly {
 -                success := delegatecall(txGas, to, add(data, 0x20), mload(data), 0, 0)
--            }
--            /* solhint-enable no-inline-assembly */
-+            // MUNGED lets just be a bit more optimistic, `execute` does nothing for `DELEGATECALL` and always returns true
++                // success := delegatecall(txGas, to, add(data, 0x20), mload(data), 0, 0)
```

Comment:
Hmm... is this supposed to be commented out?

Reply:
Nice catch! Rectified 👍🏾